File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/AppRoleFeaturePermissionModel.ts
import { AppRoleFeatureModel } from "./appRoleFeatureModel";
import { ApplicationFeaturePermissionModel } from "./applicationFeaturePermissionModel";

export interface AppRoleFeaturePermissionModel {
    rolePermissionId: number;
    appRoleFeatueId: number | null;
    featurePermissionId: number | null;
    isAllowed: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;

}

File: SofiForce.Models/Models/EntityModels/AppRoleFeaturePermissionModel.cs
using System;

namespace Models
{
    public class AppRoleFeaturePermissionModel
    {
        public int RolePermissionId { get; set; }
        public int? AppRoleFeatueId { get; set; }
        public int? FeaturePermissionId { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}
